app/src/components/Card/CardPreview/index.tsx

```diff
@@ -4,43 +4,30 @@ import { Image, Avatar, AvatarGroup } from '@mantine/core'
 import TaskPreview from '../../Task/TaskPreview'
 import { MessageIcon, UpdateIcon } from '../../icons'
 import ProjectUsers from '../../User/ProjectUsers'
+import { useMemo } from 'react'
+import { formatDistanceToNow } from 'date-fns'
```

Comment:
If you use the package, you need to install it. Btw you have an error in the build because of this 🫠

Reply:
@gainullin81 make sure date-fns and all new packages are committed in package.json